src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
```

Comment:
If we moved the register logic to `LiveServiceFacade` and expose it at `Kinvey.LiveService` then `LiveService.register()` would be exposed as a public API too along with `User.register()`. We should not expose both to prevent confusion.